starters/quickstart-nextjs-starter/package.json

```diff
@@ -8,18 +8,25 @@
     "build": "next build",
     "start": "next start -p 5000",
     "test": "jest",
+    "next:info": "next info",
     "telemetry:disable": "next telemetry disable",
     "telemetry:status": "next telemetry status",
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
-    "type-check": "tsc"
+    "type-check": "tsc",
+    "docker:build": "docker build -f Dockerfile -t ffleandro/entando-webui-quickstart-nextjs-starter:latest .",
```

Comment:
I think we need to update the docker image tag here instead of using your personal dockerhub account.